client/src/components/modals/nr-not-required.vue

```diff
@@ -1,29 +1,35 @@
 <template>
   <!-- NB: dummy modal attach point so unit tests can see it -->
-  <v-dialog v-model="showModal" max-width="40%" attach="">
-    <v-card class="pa-9">
-      <v-card-text class="h3">You don't need a Name Request if...</v-card-text>
+  <v-dialog v-model="showModal" width="50rem" attach="">
+    <v-card class="pa-9 nr-not-required-dialog">
+      <v-card-title>
+        <span>
+          <slot name="title">You don't need a Name Request if...</slot>
```

Comment:
I don't get it. Why do you need to specify the slot? What's wrong with the old code?

Reply:
h3 style is not the one to be used. If you put the plain text under v-card-title it is not picking the style that is added in the overrides stylesheet.

Reply:
Shouldn't this modal look like the other modals? Please compare with, say, the Cancel Name Request modal.

Reply:
Updated to be in sync with cancel and receipts modal